### PR TITLE
Clarify getting started section of README

### PR DESCRIPTION
**Pull request by iggy**
**Summary**

* There seems to be a common misconception in the Firecracker Slack
channel that you have to start off by building Firecracker when there
are binaries ready for download that don't require the whole docker
setup. Bit me and I've seen some other people in the Slack do the same
thing
* Hopefully this gives a useful pointer to where to look if you just
want to run Firecracker

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ sourced under [Apache version 2.0](LICENSE).
 To read more about Firecracker, check out
 [firecracker-microvm.io](https://firecracker-microvm.github.io).
 
-## Getting Started
+## Getting Started Building Firecracker
+
+Note: If you just want to run Firecracker see the Releases section below.
```
Comment (andreeaflorescu) [inline README.md]: I would rephrase it to
```
To get started with Firecracker, download the latest 
[release](https://github.com/firecracker-microvm/firecracker/releases) binaries or build it from source.
```

File: README.md [hunk 2]
```diff
@@ -31,7 +31,9 @@ sourced under [Apache version 2.0](LICENSE).
...
 
 You can build Firecracker on any system that has Docker running (we use a
 development container). The simple steps to get & build Firecracker are:
```
Comment (andreeaflorescu) [inline README.md]: This can also be changed to:
```
You can build Firecracker on any system that has Docker running (we use a
development container) as follows:
```

Comment (iggy) [reply README.md]: Updated.

File: README.md (hunk 1 above)
Comment (iggy) [reply README.md]: Updated.

**Comment by andreeaflorescu** [comment]
LGTM, please squash your commits into one and sign it so we can merge it.

**Comment by andreeaflorescu** [comment]
@raduweiss I think the current form from this PR is better. The simplest way to get started with firecracker is downloading the binaries.

**Comment by raduweiss** [comment]
Never mind, I looks like I actually had a tab opened with an older version of the PR :) Looking at it's current form, I'm on-board.
